tutorials/time-freq/10_spectrum_class.py

```diff
@@ -0,0 +1,172 @@
+    event_dict = {'auditory/left': 1, 'auditory/right': 2, 'visual/left': 3,
+                  'visual/right': 4}
+    epochs = mne.Epochs(raw, events, tmin=-0.3, tmax=0.7, event_id=event_dict,
+                        preload=True)
+epo_spectrum = epochs.compute_psd()
+psds, freqs = epo_spectrum.get_data(return_freqs=True)
+print(f'\nPSDs shape: {psds.shape}, freqs shape: {freqs.shape}')
+epo_spectrum
+
+# %%
+# Additionally, both :class:`~mne.time_frequency.Spectrum` and
+# :class:`~mne.time_frequency.EpochsSpectrum` have ``__getitem__`` methods,
+# meaning their data can be accessed by square-bracket indexing. For
+# :class:`~mne.time_frequency.Spectrum` objects (computed from
+# :class:`~mne.io.Raw` or :class:`~mne.Evoked` data), the indexing works
+# similar to a :class:`~mne.io.Raw` object or a
+# :class:`NumPy array<numpy.ndarray>`:
+
+evoked = epochs['auditory'].average()
+evk_spectrum = evoked.compute_psd()
+# the first 3 frequency bins for the first 4 channels:
+print(evk_spectrum[:4, :3])
+
+# %%
+# .. hint::
+#    :class: sidebar
+#
+#    If the original :class:`~mne.Epochs` object had a metadata dataframe
+#    attached, the derived :class:`~mne.time_frequency.EpochsSpectrum` will
+#    inherit that metadata and will hence also support subselecting epochs via
+#    :ref:`Pandas query strings <pandas:indexing.query>`.
+#
+# In contrast, the :class:`~mne.time_frequency.EpochsSpectrum` has indexing
+# similar to :class:`~mne.Epochs` objects: you can use string values to select
+# spectral estimates for specific epochs based on their condition names, and
+# what you get back is a new instance of
+# :class:`~mne.time_frequency.EpochsSpectrum` rather than a
+# :class:`NumPy array<numpy.ndarray>` of the data values. Selection via
+# :term:`hierarchical event descriptors` (HEDs) is also possible:
+
+# get both "visual/left" and "visual/right" epochs:
+epo_spectrum['visual']
+
+# %%
+# Visualizing Spectrum objects
+# ----------------------------
+#
+# Both :class:`~mne.time_frequency.Spectrum` and
+# :class:`~mne.time_frequency.EpochsSpectrum` objects have plotting methods
+# :meth:`~mne.time_frequency.Spectrum.plot` (frequency × power),
+# :meth:`~mne.time_frequency.Spectrum.plot_topo` (frequency × power separately
+# for each sensor), and :meth:`~mne.time_frequency.Spectrum.plot_topomap`
+# (interpolated scalp topography of power, in specific frequency bands). A few
+# plot options are demonstrated below; see the docstrings for full details.
+
+evk_spectrum.plot()
+evk_spectrum.plot_topo(color='k', fig_facecolor='w', axis_facecolor='w')
+
+# %%
+evk_spectrum.plot_topomap(ch_type='eeg', agg_fun=np.median)
+
+# %%
+# Migrating legacy code
+# ---------------------
+#
+# Below is a quick-reference table of equivalent code from before and after the
+# introduction of the :class:`~mne.time_frequency.Spectrum` and
+# :class:`~mne.time_frequency.EpochsSpectrum` classes.
+#
+# .. table:: Quick reference for common Spectral class actions
```

Comment:
The table title is off-center for me

https://output.circle-artifacts.com/output/job/b0cf67fb-c753-4c24-9ec2-0e9877d45f9e/artifacts/0/dev/auto_tutorials/time-freq/10_spectrum_class.html#id4

<img width="810" alt="Screen Shot 2022-08-25 at 9 29 24 AM" src="https://user-images.githubusercontent.com/2365790/186615619-f8dc4d7a-62ed-487e-aa4b-fe4fc8750da7.png">


Can you either convert this to an issue, see if the fix is easy, or open an upstream pydata-sphinx-theme issue?

Reply:
it scrolls.